Tools/MG Matrix.swift

```diff
@@ -17,6 +17,7 @@ protocol NumericType:
 
 extension Double: NumericType
 {
+    // FIXME: Compiler warns about function call causing infinite recursion - MG
```

Comment:
Are you planning on actually fixing these warnings in this PR?

Reply:
I don't believe this is my code.  I can have a look at it, though.